File: C#/Basic logic/A/A12.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Cšarp_příklady
{
    internal class A12
    {
        static void Mainx(string[] args)
        {

            Console.WriteLine("zadej průměr bazénu (m)");
            int prumer =int.Parse(Console.ReadLine());
            Console.WriteLine("zadej výšku bazénu (m)");
            int vyska =int.Parse(Console.ReadLine());
            double vypocet = Math.PI * Math.Pow(prumer/2, 2) * vyska;
            Console.WriteLine("objem bazénu je: " + Math.Round(vypocet,3) + "m2");

        }

    }
}

File: Java/Basics To Java/src/A/A12.java
package A;

import java.util.Scanner;

public class A12 {

	public static void mainx(String[] args) {
		Scanner sc = new Scanner(System.in);

		System.out.println("jaký je prùmìr tvojeho bazénu?");
		int prumer = sc.nextInt();
		System.out.println("zadej výšku bazénu");
		int vyska = sc.nextInt();
		float vypocet = (float) (Math.PI * Math.pow(prumer / 2, 2) * vyska);
		System.out.printf("objem bazénu je %.3f m3", vypocet);
	}
}
